ocs_ci/ocs/utils.py

```diff
@@ -29,6 +30,8 @@
 from ocs_ci.utility.retry import retry
 from ocs_ci.utility.utils import create_directory_path, mirror_image, run_cmd
 
+
+CEPH_KEYRING_FILE = '/etc/ceph/keyring2'
```

Comment:
Dont we need the file name convention ```/etc/ceph/<cluster>.client.admin.keyring``` ?

Reply:
/etc/ceph/keyring should work as well( I will fix the '2' string), it supports few formats without client/username as well.